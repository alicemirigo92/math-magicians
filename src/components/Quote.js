import style from './Quote.module.css';

const Quote = () => (
  <section className={style.container}>
    <p className={style.p}>
      Mathematics is not about numbers, equations, computations, or algorithms:
      It is about understanding ---William Paul Thurston
    </p>
  </section>
);

export default Quote;
