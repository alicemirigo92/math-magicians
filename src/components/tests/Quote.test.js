import TestRenderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Quote from '../Quote';

describe('Quote Page test', () => {
  it('Whether Quote page renders', () => {
    const tree = TestRenderer
      .create(
        <BrowserRouter>
          <Quote />
        </BrowserRouter>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
