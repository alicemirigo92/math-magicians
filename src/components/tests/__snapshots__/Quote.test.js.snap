// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote Page test Whether Quote page renders 1`] = `
<section
  className="container"
>
  <p
    className="p"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: It is about understanding ---William Paul Thurston
  </p>
</section>
`;
